DEVELOPERS.md

```diff
@@ -34,13 +34,15 @@ machine:
   to be installed and included in your
   [PATH](http://docs.oracle.com/javase/tutorial/essential/environment/paths.html) variable.
 
-* [Grunt](http://gruntjs.com): We use Grunt as our build system. Install the grunt command-line tool
-  globally with:
+* [Grunt](http://gruntjs.com): We use Grunt as our build system. We're using it as a local dependencies but if you prefer to use the command-line tool
```

Comment:
Sorry to bother you so long with this :D but I feel like we can be more precise here. How about 

> We're using it as a local dependency, but you can also add the grunt command-line tool globally, which allows you to leave out the `yarn` prefix for all our grunt commands:

Then you can remove the note below (which makes it sound like you can only use global or local grunt, but you can still use yarn grunt ... if you want, even with global grunt-cli).

Reply:
No need to feel sorry, I appreciate the feedback! I do think it reads better the way u propose, let me update it!

Reply:
I will merge  when you've updated this part 👍

Reply:
Done 😄